bpmn2stamp-console/src/test/java/cz/cvut/kbss/bpmn2stamp/console/ConsoleRunnerTest.java

```diff
@@ -2,28 +2,75 @@
 
 import org.apache.commons.cli.ParseException;
 import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+
+import static org.mockito.Matchers.*;
 
 public class ConsoleRunnerTest {
 
-//	@Test
-	public void mainTest() {
-		// TODO use real files or use mockito
-		Assertions.assertThatThrownBy(() -> {
-			new ConsoleRunner().run(new String[]{"-t", "stamp", "-iri", "testBaseIri", "-ibpmn", "testBpmnInput", "-iorg", "testOrgInput", "-iam", "testActorInput", "-out", "testOutput"});
-		}).isNotInstanceOf(ParseException.class);
+	private Configuration config;
+	private NoneTypeArgsProcessor noneTypeArgsProcessor;
+	private StampTypeArgsProcessor stampTypeArgsProcessor;
+	private BboTypeArgsProcessor bboTypeArgsProcessor;
+	private StampFromBboTypeArgsProcessor stampFromBboTypeArgsProcessor;
+	private Bpmn2StampConverterService converterService;
+	
+	@BeforeEach
+	public void setUp() throws IOException {
+		this.config = Configuration.getInstance();
+//		this.noneTypeArgsProcessor = Mockito.mock(NoneTypeArgsProcessor.class);
+		this.noneTypeArgsProcessor = new NoneTypeArgsProcessor();
+		this.stampTypeArgsProcessor = Mockito.mock(StampTypeArgsProcessor.class);
+		this.bboTypeArgsProcessor = Mockito.mock(BboTypeArgsProcessor.class);
+		this.stampFromBboTypeArgsProcessor = Mockito.mock(StampFromBboTypeArgsProcessor.class);
+		this.converterService = Mockito.mock(Bpmn2StampConverterService.class);
+	}
 
-		Assertions.assertThatThrownBy(() -> {
-			new ConsoleRunner().run(new String[]{"-t", "bbo", "-iri", "testBaseIri", "-ibpmn", "testBpmnInput", "-iorg", "testOrgInput", "-iam", "testActorInput", "-out", "testOutput"});
-		}).isNotInstanceOf(ParseException.class);
+	@Test
+	public void mainTest() throws ParseException {
```

Comment:
Not sure what this test supposed to do

Reply:
This is unfinished WIP test, finished in the following commit